silkshopy-ecommerce/src/backend/db/users.js

```diff
@@ -13,7 +13,7 @@ export const users = [
     firstName: "Adarsh",
     lastName: "Balika",
     email: "adarshbalika@gmail.com",
-    password: bcyrpt.hashSync("adarshBalika123", 5),
+    password: "adarshBalika123",
```

Comment:
Don't remove encrypted password, you should have your passwords encypted only.

Reply:
yes , i will add that back